cmd/monaco/support/archive.go

```diff
@@ -19,18 +19,19 @@ package support
 import (
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/timeutils"
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/trafficlogs"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/zip"
 	"github.com/spf13/afero"
 	"path"
 )
 
+var SupportArchive bool // TODO: this needs to be refactored :_)
+
 func Archive(fs afero.Fs) error {
 	timeAnchorStr := timeutils.TimeAnchor().Format("20060102-150405")
 	zipFileName := "support-archive-" + timeAnchorStr + ".zip"
 	files := []string{
-		path.Join(log.LogDirectory, timeAnchorStr) + "-" + trafficlogs.RequestLog.FileName(),
-		path.Join(log.LogDirectory, timeAnchorStr) + "-" + trafficlogs.ResponseLog.FileName(),
+		path.Join(log.LogDirectory, timeutils.TimeAnchor().Format("20060102-150405")+"-"+"req.log"),
```

Comment:
it might still make sense to expose the filenames/filename format as a constant from trafficlogs I think? if the name changes in the logger, this breaks

Reply:
absolutely